x/gravity/keeper/keeper.go

```diff
@@ -239,3 +244,78 @@ func (gsg GravityStoreGetter) Store(ctx context.Context) storetypes.KVStore {
 func (k Keeper) GetAuthority() string {
 	return k.authority
 }
+
+func (k Keeper) BridgeTax(ctx context.Context) (*types.BridgeTax, error) {
+	var tax types.BridgeTax
+	val := k.GetStore(ctx, types.StoreModulePrefix).Get(types.BridgeTaxKey)
+
+	if len(val) == 0 {
+		// We have no bridge tax settings
+		return nil, keeperutil.ErrNotFound.Format(&tax, hex.EncodeToString(types.BridgeTaxKey))
```

Comment:
To be honest, I usually find static Errors to be enough instead of those superficial error wraps, as they make it much harder to compare against error types.

But it's not a coding style decision, and we have both approaches throughout the code base, so pick your flavour I guess.